CMakeLists.txt

```diff
@@ -56,8 +56,15 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}/external/sha256/include)
 add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/external/tinf)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/external/tinf/include)
 
+# Add Monocypher library
```

Comment:
Could we add options for the (ON/OFF) for the software crypto?

Reply:
Done.